provider/cmd/pulumi-resource-vault/bridge-metadata.json

```diff
@@ -2189,7 +2221,20 @@
             }
         }
     },
-    "auto-settings": {},
+    "auto-settings": {
+        "datasources": {
+            "vault_namespace": {
+                "renames": [
+                    "vault:namespace/get:get"
+                ]
+            },
+            "vault_namespaces": {
+                "renames": [
+                    "vault:namespace/getS:getS"
+                ]
+            }
+        }
+    },
     "renames": {
```

Comment:
Are these renames the issue?

Reply:
Probably. I regenerated everything after resetting `bridge-metadata.json` and SDKs seemed to build.